src/app/lib/providers/opensubtitles.js

```diff
@@ -29,11 +29,15 @@
         for (var lang in data) {
             data[lang] = data[lang].url;
         }
-        return Common.sanitize(data);
+        return {subtitle: Common.sanitize(data)};
     };
 
-    OpenSubtitles.prototype.fetch = function (queryParams) {
-        queryParams.extensions = ['srt'];
+    OpenSubtitles.prototype.detail = function (id, attrs) {
```

Comment:
No. That's a bad idea. We don't search by imdb only, we use filename on dropped files and so on. Let the module handle what queries it can accept and what it can't. It was written correctly before.